src/main/jbake/content/jsf-ajax012.adoc

```diff
@@ -1,21 +1,21 @@
 type=page
 status=published
-title=Further Information about Ajax in JavaServer Faces Technology
+title=Further Information about Ajax in Jakarta Server Faces Technology
 next=jsf-advanced-cc.html
 prev=jsf-ajax011.html
 ~~~~~~
-= Further Information about Ajax in JavaServer Faces Technology
+= Further Information about Ajax in Jakarta Server Faces Technology
 
 
 [[GKSDK]][[further-information-about-ajax-in-javaserver-faces-technology]]
 
-Further Information about Ajax in JavaServer Faces Technology
--------------------------------------------------------------
+Further Information about Ajax in Jakarta Server Faces Technology
+-----------------------------------------------------------------
 
-For more information on Ajax in JavaServer Faces Technology, see
+For more information on Ajax in Jakarta Server Faces Technology, see
 
-* JavaServer Faces project website:
+* Jakarta Server Faces project website:
 +
 `https://javaserverfaces.github.io/users.html`
-* link:../javaserver-faces-2-2/jsdocs/symbols/jsf.ajax.html[JavaServer
+* link:../javaserver-faces-2-2/jsdocs/symbols/jsf.ajax.html[Jakarta Server
```

Comment:
I think this should be a link to https://jakarta.ee/specifications/faces/2.3/jsdoc.